src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1643,59 +1643,6 @@ private int GetCmdletRemovalIndex(List<CmdletInfo> cacheEntry, string PSSnapin)
 
         internal ExecutionContext Context { get; }
 
-        /// <summary>
-        /// Reads the path for the appropriate shellID from the registry.
```

Comment:
Maybe leave this? How Windows user or Domain Admins can system-wide switch from Windows PowerShell to PowerShell Core?

Reply:
I don't understand your question. This method is removed because it's not used anywhere.

Reply:
My thoughts was about side-by-side scenario but it seems not help.
Closed.